File: x/6th_python/ps6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
while True:
    Height = get_int("Height: ")
    if Height > 0 and Height < 9:
         break

for i in range(1, Height + 1):
    for j in range(Height - i):
        print(" ", end="")

    for j in range(i):
        print("#", end="")

    print("  ", end="")

    for j in range(i):
        print("#", end="")

    print()

File: x/1st_c/ps1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    do
    {
        Height = get_int("Height: ");
    }
    while (Height < 1 || Height > 8);
    for (int i = 1; i <= Height; i++)
    {
        // Print spaces
        for (int j = 0; j < Height - i; j++)
        {
            printf(" ");
        }

        // Print left side of the pyramid
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // Print spaces between the two sides
        printf("  ");

        // Print right side of the pyramid
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // Move to the next line
        printf("\n");
    }
}